src/dymaptic.GeoBlazor.Core/Components/Renderers/ColorRamps/MultipartColorRamp.cs

```diff
@@ -0,0 +1,29 @@
+﻿using dymaptic.GeoBlazor.Core.Serialization;
+using Microsoft.AspNetCore.Components;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Renderers.ColorRamps;
+
+/// <summary>
+///     Creates a color ramp for use in a raster renderer. The algorithmic color ramp is defined by specifying two colors and the
+///     algorithm used to traverse the intervening color spaces.
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-rest-support-MultipartColorRamp.html">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+public class MultipartColorRamp : ColorRamp
+{
+    public MultipartColorRamp() { }
```

Comment:
In addition to implementing the `ChilcComponent` methods, add another constructor where you could pass in the `AlgorightmicColorRamp[] colorRamps` and make the property `private set;`

Reply:
childcomponents are now included and properties now have private setters